src/components/DefineCompany/BusinessContactInfo.vue

```diff
@@ -103,7 +103,7 @@ export default class BusinessContactInfo extends Vue {
   ]
 
   private extensionRules = [
-    (v: any) => !v || (v.length >= 0 || v.length <= 3) || 'Extension is invalid'
+    (v: any) => !v || (v.length >= 0 || v.length <= 5) || 'Extension is invalid'
```

Comment:
Should this be `length >= 1 && length <= 5`?

Reply:
Not sure as this works as expected with the current rule

Reply:
As we discussed, this works currently due to the input mask that allows 0-5 digits. Therefore this 'rules' is not needed.